src/main/java/Event.java

```diff
@@ -0,0 +1,12 @@
+public class Event extends Task {
+    protected String at;
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+    }
+    @Override
+    public String toString() {
+
+        return "[E]" + super.toString() + " (at: " + at + ")";
+    }
+}
```

Comment:
Consider reformatting here and in some other classes, so that you have a newline before the function and no newline before the first statement within the function